site2/docs/deploy-bare-metal.md

```diff
@@ -103,7 +103,7 @@ Directory | Contains
 
 ## [Install Builtin Connectors (optional)]( https://pulsar.apache.org/docs/en/next/standalone/#install-builtin-connectors-optional)
 
-> Since Pulsar releases `2.1.0-incubating`, Pulsar releases a separate binary distribution, containing all the `builtin` connectors.
+> Since Pulsar release `2.1.0-incubating`, Pulsar provides a separate binary distribution, containing all the `builtin` connectors.
```

Comment:
releases, third-person singular

Reply:
The first 'releases' should be a singular noun as it is describing a single release. If that is not the intent, then the '2.1.0-incubating' should have wild cards added to indicate the set of releases. For the second 'releases', I used a different term since using words with the same name and different meanings makes reading the sentence simpler.